src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upperA; //Length of the upper base
+    private final double lowerB; // Length of the lower base
+    private final double height; // height
+    private String color;
+
+    public IsoscelesTrapezoid(double upperA, double lowerB, double height, String color) {
+        this.upperA = upperA;
+        this.lowerB = lowerB;
+        this.height = height;
+        setColor(color);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isoscales trapezoid, area: " + getArea()
```

Comment:
Use only eng in messages/code: The word 'isoscales' seems to be a typo. It should be 'isosceles'.